ivy/functional/backends/jax/experimental/statistical.py

```diff
@@ -301,12 +302,8 @@ def cummax(
     dtype: Optional[jnp.dtype] = None,
     out: Optional[JaxArray] = None,
 ) -> Tuple[JaxArray, JaxArray]:
-    if x.dtype in (jnp.bool_, jnp.float16):
-        x = x.astype(jnp.float64)
-    elif x.dtype in (jnp.int16, jnp.int8, jnp.uint8):
-        x = x.astype(jnp.int64)
-    elif x.dtype in (jnp.complex128, jnp.complex64):
-        x = jnp.real(x).astype(jnp.float64)
+    if x.dtype in (jnp.complex128, jnp.complex64):
+        x = x.real
```

Comment:
sounds good, let's make this change in the other backends too in a different PR.

Reply:
understood